src/javascript/_common/third_party_links.js

```diff
@@ -0,0 +1,50 @@
+const State            = require('./storage').State;
+const Client           = require('../app/base/client');
+const BinarySocket     = require('../app/base/socket');
+const Dialog           = require('../app/common/attach_dom/dialog');
+
+const ThirdPartyLinks = (() => {
+    const init = () => {
+        if (Client.isLoggedIn()) {
+            BinarySocket.wait('landing_company').then(() => {
+                if (State.getResponse('landing_company.financial_company.shortcode') === 'maltainvest') {
+                    document.body.addEventListener('click', clickHandler);
+                }
+            });
+        }
+    };
+
+    const clickHandler = (e) => {
+        if (!e.target) return;
+        const link_el = e.target.closest('a');
+        if (!link_el) return;
+
+        const dialog = document.querySelector('#third_party_redirect_dialog');
+        if (dialog && dialog.contains(link_el)) return;
+
+        if (isThirdPartyLink(link_el.href)) {
+            e.preventDefault();
+            Dialog.confirm({
+                id     : 'third_party_redirect_dialog',
+                // TODO: pass array of sentences once negar/show_malta_popup_mt card is merged
```

Comment:
you can leave it as string, since it's just one message. my PR handles both strings and array.

Reply:
oh, then feel free to copy the part of code from my PR and use it here as an array.